File: src/main/java/org/example/UserList.java
package org.example;
import java.util.ArrayList;
public class UserList {
    ArrayList<User> userList;

    public UserList(){
        userList = new ArrayList<User>();
    }

    public void addUser(User user){

        userList.add(user);
    }

    User getUser(int index){
        return userList.get(index);
    }

    public int getNumberOfUsers(){return 0;}


}

File: UserList.js
class UserList {
    constructor() {
        this.userList = [];
    }

    addUser(user) {
        this.userList.push(user);
    }

    getUser(index) {
        return this.userList[index];
    }

    getNumberOfUsers() {
        return this.userList.length;
    }
}

module.exports = UserList;